besu/src/main/java/org/hyperledger/besu/services/BesuPluginContextImpl.java

```diff
@@ -104,6 +107,20 @@ public void registerPlugins(final Path pluginsDir) {
     state = Lifecycle.REGISTERED;
   }
 
+  private void addPluginVersion(final BesuPlugin plugin) {
+    final Package pluginPackage = plugin.getClass().getPackage();
+    final String implTitle =
+        Optional.ofNullable(pluginPackage.getImplementationTitle())
```

Comment:
Wondering if there's a (minor) funny hole here - we request the plugin maker to correctly populate their MANIFEST - but we then go and munge the data they provide into the string we want to present (eg putting the "/v" in the middle of it). 
Do we want a plugin to report its name and version via API?

Reply:
plugin api will surely enforce version string from plugin developer. I put /v to follow the same approach as `BesuInfo.version()` method.